ambry-rest/src/test/java/com.github.ambry.rest/NettyMessageProcessorTest.java

```diff
@@ -183,7 +183,6 @@ public void requestHandleWithBadInputTest()
     channel.writeInbound(new DefaultLastHttpContent(Unpooled.wrappedBuffer(content.getBytes())));
     HttpResponse response = (HttpResponse) channel.readOutbound();
     assertEquals("Unexpected response status", HttpResponseStatus.BAD_REQUEST, response.getStatus());
-    assertFalse("Channel is still active", channel.isActive());
```

Comment:
These are BAD_REQUEST responses. We no longer close the channel when there are bad requests and the method is not POST.

Also, this check never logically belonged here since `NettyMessageProcessor` is not the one who handles keep-alive or closes channels. Those tests are now where they belong in `NettyResponseChannelTest`.